.github/workflows/push-docker-image.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: |
           package_name=$(grep "Package:" DESCRIPTION | awk '{print $NF}')
           if [ "${{ matrix.tags }}" == "latest" ]; then image_name="$package_name-release"; else image_name="$package_name-${{ matrix.tags }}"; fi
-          if [ "${{ matrix.tags }}" == "devel" ]; then R_REPOS="https://packagemanager.posit.co/cran/__linux__/focal/latest"; fi 
+          if [ "${{ matrix.tags }}" == "devel" ]; then R_REPOS="https://packagemanager.posit.co/cran/__linux__/focal/latest"; fi
```

Comment:
could we drop this since no admiral repo uses `devel` anymore?

Reply:
Indeed we do thnaks @cicdguy